core/helper/CustomLog/index.py

```
@@ -102,24 +238,57 @@ async def _async_store(self) -> None:
+                            request_method=self.request_method,
+                            request_url=self.request_url,
+                            trace_id=self.trace_id,
+                            error_code=self.error_code,
+                            error_msg=self.error_msg,
+                            extra_data=_json_safe(self.extra_data) if self.extra_data else None,
                         )
                     )
         except Exception:
-            pass  # 日志存储失败不影响主流程
+            # 日志存储失败不应影响主流程；生产环境建议接入监控告警
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)